tests/Unit/VoyagerTest.php

```diff
@@ -42,6 +42,48 @@ public function testDimmersReturnsCollectionOfConfiguredWidgetsWhichShouldBeDisp
 
         $dimmers = Voyager::dimmers();
 
-        $this->assertEquals(1, $dimmers->count());
+        $this->assertEquals(1, $dimmers[0]->count());
+    }
+
+    /**
+     * Dimmers returns an array filled with widget collections.
+     *
+     * This test ensures that each widget collection can hold up to three widgets
+     */
+    public function testDimmerCollectionContainsThreeDimmersMax()
```

Comment:
How can you test that collection contains three dimmers max if you are only passing three? :thinking: